src/main/java/pl/lodz/p/it/zzpj/botsite/web/controllers/MessageController.java

```diff
@@ -79,12 +74,12 @@ public MessageDTO getMessageByIdForCurrentUser(@PathVariable("id") Long messageI
             consumes = MediaType.APPLICATION_JSON_VALUE,
             produces = MediaType.APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.CREATED)
-    public MessageDTO addMessage(@RequestBody MessageDTO messageDTO) throws UserRetrievalException, MessageAdditionException {
+    public MessageDTO addMessage(@RequestBody MessageDTO messageDTO) throws MessageAdditionException {
         if (StringUtils.isBlank(messageDTO.getContent())) {
             throw new MessageAdditionException("Content cannot be blank.");
         }
         Message message = this.modelMapper.map(messageDTO, Message.class);
-        User user = userService.findByLogin(principalProvider.getName());
+        User user = principalProvider.getUser();
```

Comment:
A masz gwarancję, że ta encja będzie aktualna ze stanem w bazie danych? Jaka jest potrzeba tego użytkownika w pełni trzymać?

Reply:
won't do

Reply:
won't pass XD

Reply:
will have to